client/src/core/client/stream/tabs/Notifications/NotificationsPaginator.tsx

```diff
@@ -52,35 +55,93 @@ const NotificationsPaginator: FunctionComponent<Props> = (props) => {
     return null;
   }
 
+  if (props.query.notifications.edges.length === 0) {
+    return (
+      <div className={styles.none}>
+        <Localized id="notifications-youDoNotCurrentlyHaveAny">
+          You do not currently have any notifications
+        </Localized>
+      </div>
+    );
+  }
+
   return (
-    <div>
-      {props.query.notifications.edges.map(({ node }) => {
-        return (
-          <NotificationContainer
-            key={node.id}
-            notification={node}
-            viewer={props.query.viewer}
-          />
-        );
-      })}
-      {isRefetching && <Spinner />}
-      {!isRefetching && !disableLoadMore && props.relay.hasMore() && (
-        <Localized id="notifications-loadMore">
-          <Button
-            key={props.query.notifications.edges.length}
-            onClick={loadMore}
-            variant="outlined"
-            color="secondary"
-            fullWidth
-            disabled={disableLoadMore}
-            aria-controls="notifications-loadMore"
-            className={CLASSES.tabBarNotifications.loadMore}
+    <>
+      <div className={styles.methodOfRedress}>
+        {props.query.settings.dsa.methodOfRedress.method ===
+          GQLDSA_METHOD_OF_REDRESS.NONE && (
+          <Localized id="notifications-methodOfRedress-none">
+            All moderation decisions are final and cannot be appealed
+          </Localized>
+        )}
+        {props.query.settings.dsa.methodOfRedress.method ===
+          GQLDSA_METHOD_OF_REDRESS.EMAIL && (
+          <Localized
+            id="notifications-methodOfRedress-email"
+            vars={{
+              email: props.query.settings.dsa.methodOfRedress.email,
+            }}
+            elems={{
+              a: (
+                <a
+                  href={`mailto: ${props.query.settings.dsa.methodOfRedress.email}`}
+                >
+                  {props.query.settings.dsa.methodOfRedress.email}
+                </a>
+              ),
+            }}
           >
-            Load More
-          </Button>
-        </Localized>
-      )}
-    </div>
+            <span>{`To appeal a decision that appears here please contact ${props.query.settings.dsa.methodOfRedress.email}`}</span>
+          </Localized>
+        )}
+        {props.query.settings.dsa.methodOfRedress.method ===
+          GQLDSA_METHOD_OF_REDRESS.URL && (
+          <Localized
+            id="notifications-methodOfRedress-url"
+            vars={{
+              url: props.query.settings.dsa.methodOfRedress.url,
+            }}
+            elems={{
+              a: (
+                <a href={`${props.query.settings.dsa.methodOfRedress.url}`}>
```

Comment:
Just checking...should this open in a new tab?

Reply:
I've added `target="_blank"` and  `rel="noreferrer"` to these links!